packages/common/src/sqs.ts

```diff
@@ -93,6 +94,29 @@ export const sendMessage = async (
 	}
 };
 
+export const changeMessageVisibility = async (
+	client: SQSClient,
+	queueUrl: string,
+	receiptHandle: string,
+	timeoutOverride: number,
+) => {
+	const command = new ChangeMessageVisibilityCommand({
+		QueueUrl: queueUrl,
+		VisibilityTimeout: timeoutOverride,
+		ReceiptHandle: receiptHandle,
+	});
+
+	try {
+		await client.send(command);
+		console.log(
+			`Successfully updated the VisibilityTimeout of the message to ${timeoutOverride}`,
+		);
+	} catch (error) {
+		const errorMsg = 'Failed to update VisibilityTimeout for message';
+		console.error(errorMsg, error);
```

Comment:
is it worth throwing the error here? I think if we fail to update the visibility timeout we should abandon the transcription rather than carrying on and risk it being done by another machine